### PR TITLE
added a way to include any images displayed on the pages to the googl…

### PR DESCRIPTION
**Pull request by fonsekaean**
…e site maps and updated the XSL to show them on the screen.

### CODE REVIEW

File: templates/xml-sitemap.xsl [hunk 1]
```diff
@@ -61,7 +88,7 @@
               </xsl:for-each>
             </tbody>
           </table>
-          <p id="Footer" class="expl">Generated by the SilverStripe
+          <p id="Footer" class="expl">Generated by the SilverStripe dd
```
Comment (dhensby) [inline templates/xml-sitemap.xsl]: was this intentional?

File: code/extensions/GoogleSitemapSiteTreeExtension.php [hunk 2]
```diff
@@ -122,4 +122,32 @@ public function getGooglePriority()
             return (is_numeric($priority) && $priority <= 1.0) ? $priority : 0.5;
         }
     }
+
+    public function ImagesForSitemap()
+    {
+        $list = new ArrayList();
+
+        foreach ($this->owner->hasOne() as $field => $type) {
+            if(is_a(singleton($type), 'Image')) {
```
Comment (dhensby) [inline code/extensions/GoogleSitemapSiteTreeExtension.php]: `singleton($type) instanceof Image` would be better here

File: templates/xml-sitemap.xsl (hunk 1 above)
Comment (fonsekaean) [inline templates/xml-sitemap.xsl]: woops.

File: code/extensions/GoogleSitemapSiteTreeExtension.php (hunk 2 above)
Comment (fonsekaean) [inline code/extensions/GoogleSitemapSiteTreeExtension.php]: sure we can do this

**Comment by dhensby** [comment]
@wilr looks like your travis builds are broken

**Comment by wilr** [comment]
@fonsekaean Could you rebase on the latest release and add a test for this. Thanks

**Comment by dhensby** [comment]
I've gone and rebased this, I haven't added a test, though

**Pull request merged**